### ISSUE TITLE
MIssing ore?

### ISSUE THREAD

**Issue by CameraGuy** [adam9899/Issues#158]
Mek ore does not seem to be spawning in the world? I am using a digi miner and have found just about nothing beyond vanilla stuff! Have I missed something?

**Comment by ByteTemplar** [comment]
Yes, take a look at Quest book, but some ore spawns are gated to different dimensions to slow down progression. For example osmium is in Beneath dimension...

Industrial foregoing laser miner can access most ore in Overworld though ;)  Just not the most fastest thing, but can have multiple ones if needed and use different lenses.
